libs/core/langchain_core/tracers/langchain_v1.py

```diff
@@ -60,29 +61,30 @@ def _convert_to_v1_run(self, run: Run) -> Union[LLMRun, ChainRun, ToolRun]:
             else:
                 raise ValueError("No prompts found in LLM run inputs")
             return LLMRun(
-                uuid=str(run.id) if run.id else None,
+                uuid=str(run.id) if run.id else "",
```

Comment:
Is this safe? Is downstream code ever assuming this is a None?

Reply:
yea should maybe type ignore instead. cc @hinthornw any idea if None is expected downstream?

Reply:
We should just delete this file entirely. It's not used anymore. @agola11 had a PR out to remove at some point that must have never been merged